### PR TITLE
Added basic general release process diagram

### PR DESCRIPTION
**Pull request by manicki**

### CODE REVIEW

File: diagrams/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+`docker build -t diagrams .`
+`docker run -it --rm -v "$PWD:/app" -w /app  node:14 npm install`
+
+Generate a SVG representation of the mermaid diagram with:
+
+`docker run -it --rm -v "$PWD:/app" -w /app  diagrams node_modules/.bin/mmdc -i input.mmd -o output.svg`
+
+If you get a Chrome error `Running as root without --no-sandbox is not supported` you might need to use the `--no-sandbox` option of the Puppeteer.
+
+`ocker run -it --rm -v "$PWD:/app" -w /app  diagrams node_modules/.bin/mmdc -p puppeteer-config.json -i input.mmd -o output.svg`
```
Comment (mariushoch) [inline diagrams/README.md:17]: typo (ocker <> docker)

**Comment by mariushoch** [comment]
Having the README in the subfolder (only) might bury this a bit (hard-ish to find out it exists), other than that, this sounds like a good idea to me.

**Review by toban — APPROVED** [review]
LGTM and like the chart syntax. We should generate these as artifacts on build!

**Comment by manicki** [comment]
FWIW `input.mmd` was a silly way of saying this is a generic command (based on the unjustified assumption that there will be more diagram files than just this very high level one)

**Pull request merged**